src/test/ui/issues/issue-45829/import-twice.stderr

```diff
@@ -2,15 +2,13 @@ error[E0252]: the name `A` is defined multiple times
   --> $DIR/import-twice.rs:6:14
    |
 LL | use foo::{A, A};
-   |           -  ^ `A` reimported here
-   |           |
+   |           ---^
+   |           || |
+   |           || `A` reimported here
+   |           |help: remove unnecessary import
    |           previous import of the type `A` here
```

Comment:
These are getting crowded, but will need to be handled at the emitter level, seeing that too many things are overlapping and separating them into different windows (like it used to do with the `as` suggestion). No action item on your part.

Reply:
Filed https://github.com/rust-lang/rust/issues/58032/